ext/README.md

```diff
@@ -18,7 +18,7 @@ __NOTE__: The build procedures only need to be performed for driver development
 
 ## Compatibility
 
-* PHP 5.6, PHP 7.0, and PHP 7.1
+* PHP 5.6, PHP 7.0, PHP 7.1 and PHP 8.1
```

Comment:
we could be running a CI matrix in order to assert it.
maybe we should enable the GH CI in this repository because atm the CI relies on Travis, wdyt ?

Reply:
Let's just drop older versions support for now